src/helpers/s3Helper.js

```diff
@@ -7,20 +7,31 @@ const s3 = new AWS.S3();
 const s3Bucket = process.env.TRYNAPI_S3_BUCKET || "orion-vehicles";
 console.log(`Reading state from s3://${s3Bucket}`);
 
+function convertTZ(date, tzString) {
+  return new Date((typeof date === "string" ? new Date(date) : date).toLocaleString("en-US", {timeZone: tzString}));
+}
+
 /*
- * Gets bucket prefix at the minute-level
+ * Gets bucket prefix at the hour-level
+ * Note to Jesse - I changed the bucket structure
+ * when we switch to a new bucket (code for PDX owned)
+ * we could switch it back. For now, it's not great
+ * but I think it's not the source of the problem.
+ * see getVehiclePaths - I think the function is
+ * not as fast as it could be but it works
  * @param agencyId - String
  * @param currentTime - Number
  * @return prefix - String
  */
-function getBucketMinutePrefix(agencyId, currentTime) {
+function getBucketHourPrefix(agencyId, currentTime) {
   const currentDateTime = new Date(Number(currentTime * 1000));
-  const year = currentDateTime.getUTCFullYear();
-  const month = currentDateTime.getUTCMonth()+1;
-  const day = currentDateTime.getUTCDate();
-  const hour = currentDateTime.getUTCHours();
-  const minute = currentDateTime.getUTCMinutes();
-  return `${agencyId}/${year}/${month}/${day}/${hour}/${minute}/`;
+  const pacificDateTime = convertTZ(currentDateTime, 'America/Los_Angeles');
```

Comment:
Using America/Los_Angeles as the time zone could cause issues related to daylight savings since when switching from PDT to PST in the fall the there would be two hours with the same key prefix.

Using America/Los_Angeles would also be confusing when deploying opentransit-state-api for transit agencies that are not in Pacific time. It is also possible that a single instance of opentransit-state-api could support transit agencies in multiple time zones. It would probably be simpler to keep the S3 keys in UTC.